src/deprecated.jl

```diff
@@ -1,18 +1,23 @@
-using Base.depwarn
+import Base: @deprecate, depwarn
```

Comment:
`import` is not needed here as we don't extend these methods. Also, better have this in a separate commit.